server/index.js

```diff
@@ -1,11 +1,30 @@
-if (process.env !== 'production') {
+if (process.env.NODE_ENV !== 'production') {
   require('dotenv').config();
 };
+
+
 const bodyParser = require('body-parser');
-const express = require('express');
+const express = require('express'),
+  app = express(),
+  session = require('express-session');
```

Comment:
IMHO ... I've been using the [airbnb style guide](https://github.com/airbnb/javascript#variables) with eslint last few projects and comma separated variable declaration is considered bad practice. I don't really want to use prettier but I think we should say what style guide we are going to follow to help keep code consistent

Reply:
agree

Reply:
Airbnb is great for readability purpose